packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/RetroTopic.tsx

```diff
@@ -75,6 +133,7 @@ const RetroTopic = (props: Props) => {
       : `See ${commentCount} ${plural(commentCount, 'Comment')}`
   const commentLinkStyle = commentCount === 0 ? noCommentLinkStyle : someCommentsLinkStyle
   const corsOptions = isEmail ? EMAIL_CORS_OPTIONS : APP_CORS_OPTIONS
+  const showSummary = viewerMeetingMember?.user.featureFlags.aiSummary && summary
```

Comment:
+1 because we generate the summary conditionally, I think there is no need to check for the feature flag here again.